src/bundle/Resources/views/themes/admin/matrix_fieldtype/content_edit_form_fields.html.twig

```diff
@@ -14,35 +14,38 @@
             }
         }]) %}
     {% endfor %}
+    {% set head_cols = head_cols|merge([{}]) %}
 
     {% embed '@ezdesign/ui/component/table/table.html.twig' with {
         headline: form.vars.label,
         head_cols: head_cols,
+        class: 'ibexa-table--not-striped',
         table_body_attr: {
             'data-minimum-rows': minimum_rows,
             'data-next-index': form.entries.children|length ? max(form.entries.children|keys) + 1 : 0,
         },
     } %}
         {% block header %}
             {% embed '@ezdesign/ui/component/table/table_header.html.twig' %}
+                {% block headline %}{% endblock %}
```

Comment:
I guess you achieve the same effect by just removing the `headline` property from the 20th line.